### DISCUSSION TITLE
I pushed to "Master"

### DISCUSSION THREAD

**Discussion by markusschloesser** [markusschloesser/MackieC4_P3#75] [General]
@BilldarBagdar 
after working with your latest encodermaphistory branch for a couple of days, I decided to push to master, as the status is very very good (imho).
I still get the occasional error (will open separate issues for those), but so do other scripts. Do you wanna have the honor of making a "release"? 
I am very happy with what we have achieved and I would be nowhere without your tremendous amount of work, patience and explaining. I am very grateful for that! ☺☺☺

**Comment by BilldarBagdar** [comment]
Fantastic!  I agree it's time for an update to Master, and I had not thought about a release too but yes, I would agree.  It's stable enough now in 10, and 11, that we could probably support a release.  (I mean we can always ghost support requests or whatever anyway, that happens all the time, right?  I just mean I would "stand behind" this product as a release.)

Yes, I can take on the task of preparing a "release branch".  The main thing I want to do for "release" is silence 99% or more of the "debug logging" we added (me mostly I think).

**Reply by markusschloesser** [reply]
ok great! 
re silencing: still helpful for us, so can we leave it in and just let it run against dev/null? ;-) like disable the logging function?
